nss-day-cohort-48/giffygram-altavista-giffygram
# Jf post form

#### Changes Made

1. Got post list to render on DOM
2. Got the form to post a new meme to the feed.
3. Moved the PostList folder to the Post folder from the feed folder.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout jf-post-form
   ```
2. Open a new Terminal and open the giffygram repo (make sure it's named 'giffygram')
3. Run `./serve-giffygram` to serve the code.
4. Test app functionality.
   > When you enter the post info when you submit you should see your post appear.
   >When you select cancel the text should clear.
   > PostList module should appear in the Post folder.
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
